akka-sample-javaslang/src/main/java/sample/javaslang/Master.java

```diff
@@ -0,0 +1,54 @@
+package sample.javaslang;
+
+import akka.actor.Props;
+import akka.actor.UntypedAbstractActor;
+import javaslang.collection.List;
+import javaslang.collection.Stream;
+
+import static sample.javaslang.Minion_StoppedPatterns.*;
+
+import static javaslang.API.*;
+import static javaslang.Predicates.*;
+
+public class Master extends UntypedAbstractActor {
+
+  private final Maze maze;
+
+  private List<Minion.Stopped> results = List.empty();
+
+  public Master(Maze maze) {
+    this.maze = maze;
+  }
+
+  public void onReceive(Object message) {
```

Comment:
Would be very cool to use it in the `public Receive createReceive() {` though.
We could do a mini library with those wrappers, I think it could be nice - separate from akka core though